coral-component-list/src/scripts/SelectList.js

```diff
@@ -431,13 +404,39 @@ class SelectList extends BaseComponent(HTMLElement) {
    @private
    */
   _resetTabTarget(forceFirst = false) {
-    let items = this.items._getAllSelected().filter(item => !item.hasAttribute('hidden'));
+    let items = this.items._getAllSelected().filter(item => !item.hasAttribute('hidden') && item.offsetParent);
 
     if (items.length === 0) {
-      items = this.items._getSelectableItems().filter(item => !item.hasAttribute('hidden'));
+      items = this.items._getSelectableItems().filter(item => !item.hasAttribute('hidden') && item.offsetParent);
     }
     
-    this._tabTarget = forceFirst ? items[0] : (items.find(item => item.tabIndex === 0) || items[0]); 
+    this._tabTarget = forceFirst ? items[0] : (items.find(item => item.tabIndex === 0) || items[0]);
+
+    if (this.groups && this._groups.getAll().length > 0) {
```

Comment:
Can the _tabTarget be outside of groups?
If not then  in the code below line-415 will always set the firstFocusable tabindex to -1  and we won't need to set it to 0 .

What is the intention of code below line-415?
AFAIK we want to set tabindex=-1 on all but first item across all groups, so only one item can be focused across all the groups.
Doesn't line-227 do the same thing? (set tabindex = -1 on all but one item)